lib/pQueue.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Might want to add these `lib/` files to your `.gitignore` to lower the amount of noise in your repo.